source/security/index.html.md.erb

```diff
@@ -95,12 +95,13 @@ data must comply with the [Payment Card Industry Data Security
 Standards](https://www.pcisecuritystandards.org/document_library?category=pcidss&document=pci_dss) (PCI DSS).
 
 GOV.UK Pay is certified as fully compliant as a Level 1 Service Provider with
-PCI DSS version 3.2.1. All GOV.UK Pay partners must be compliant with PCI DSS,
-and must validate their compliance annually.
+PCI DSS version 4.0. 
 
-A Qualified Security Assessor will audit GOV.UK Pay against PCI DSS v4.0 in summer 2024. After this audit, we'll update all relevant PCI DSS documentation. 
+All GOV.UK Pay partners and any services that take MOTO payments through GOV.UK Pay must comply with PCI DSS v4.0 by 31 March 2025.
 
-You may be asked to provide certain information from GOV.UK Pay as part of your own PCI DSS compliance process. Some of this information may not be available until we've completed our PCI DSS v4.0 transition work. This should not affect your ability to comply with PCI DSS v4.0 because there is a recognised transition period.
+If your service takes MOTO payments, you should familiarise yourself with the [the changes from PCI DSS v3.2.1 to v4.0](https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v3-2-1-to-v4-0-Summary-of-Changes-r2.pdf).
+
+You may be asked to provide certain information from GOV.UK Pay as part of your own PCI DSS compliance process. You can see our PCI DSS Attestation of Compliance by signing into [the GOV.UK Pay admin tool](https://selfservice.pymnt.uk/my-services) and selecting Attestation of Compliance for PCI in the footer.
```

Comment:
Does the link to pymn.uk automagically update to production urls in the future? otherwise this will be the wrong link.